blockchain/state_transition.go

```diff
@@ -135,7 +135,16 @@ type kerror struct {
 
 // NewStateTransition initialises and returns a new state transition object.
 func NewStateTransition(evm *vm.EVM, msg Message) *StateTransition {
-	effectiveGasPrice := msg.EffectiveGasPrice(evm.BaseFee)
+	var effectiveGasPrice *big.Int
+	if !evm.ChainConfig().IsKIP71ForkEnabled(evm.BlockNumber) {
+		// TODO-klaytn it use calculation algorithm of ethereum
+		// We have used it just like mock function. It returns governance unitprice
+		// after hard fork, we need to consider new getEffectiveGasPrice API that operates differently
+		effectiveGasPrice = msg.EffectiveGasPrice(evm.BaseFee)
```

Comment:
Any difference between evm.BaseFee and param.basefee?

Reply:
Please look at the above review

Reply:
We have both `evm.BaseFee` and `params.BaseFee` to keep compatibility with ethereum https://github.com/klaytn/klaytn/pull/1065. 
`evm.BaseFee` is for opcode `opBaseFee()` that is used in evm and params.BaseFee is just for the constant value.
However It needs to be hard forked from now on